README.md

```diff
@@ -24,9 +24,22 @@ Default Docker image is [yegor256/rultor](https://registry.hub.docker.com/u/yego
 
 TBD... _product statement_
 
-## What Problem Does Rultor Solve?
+## What Problems Does Rultor Solve?
 
-TBD... _stakeholders and needs_
+Automated build scripts have been around for some time. Rultor attempts to
+tackle the problems those scripts do not.
```

Comment:
I did describe them, although not explicitly.

+ If you run these scripts without a clean container, the number of external variables that may affect your build is much higher. Rultor manages the containers for you.
+ Rultor makes logs easily accessible, right where you called it. If you need CI logs, you usually have to go somewhere else to fetch them. Not to mention that they may be dropped after some time.
+ Rultor makes DevOps more human. Instead of invoking a script (that may not be a bash script, and a non-developer may have no clue about what to do), one can use plain English to instruct Rultor what to do. You program it only once (in .rultor.yml) and then it encapsulates what it is doing under the hood.

The paragraphs 2, 3, and 4 are essentially these points.